tests/utilities/test_app.py

```diff
@@ -25,13 +25,15 @@
 
 _SYNC_START_SEQUENCE = TERMINAL_MODES_ANSI_SEQUENCES["sync_start"]
 
+_DEFAULT_SIZE = Size(20, 10)
```

Comment:
for some tests, such as this key bindings new one, we _really_ don't care about the size of the app :slightly_smiling_face:  - let's make this size optional!